lib/sms_api.rb

```diff
@@ -6,6 +6,15 @@ def initialize
     @client = Savon.client(wsdl: url, convert_request_keys_to: :camelcase)
   end
 
+  def sms_deliver(phone, code)
+    return false unless end_point_available?
+
+    response = client.call(:sms_text_submit, message: request(phone, code))
+    success?(response)
+  end
+
+  private
```

Comment:
While I think these changes are useful, I'd rather have the code as similar to CONSUL's code as possible so it's easier to deal with conflicts when upgrading :thinking:. But if you think differently, we can include this change :wink:.

Reply:
Done